src/components/gameScene/GameScreenLayout.tsx

```diff
@@ -6,8 +6,8 @@ import { useAppSelector } from "@app/Store";
 import EmptyAdminPlace from "@app/components/gameScene/gameMasterComp/EmptyAdminPlace";
 
 export default function GameScreenLayout() {
-  const gameLobby = useAppSelector((s) => s.userInfo.gameLobby);
-  const { createdBy, me } = useAppSelector((s) => s.userInfo);
+
+  const { lobby, amMaster } = useAppSelector((s) => s.gameState);
```

Comment:
Please just use select with the full path of the data that you will actually use next.
Example
`IsOccupiedMaster = useAppSelector((s) => s.gameState.lobby?.Master.IsOccupied)` instead of
`const { lobby, amMaster } = useAppSelector((s) => s.gameState);`

This can help avoid unnecessary component re-renders

Reply:
Oh, i see. Good point. I'm on it